frontend/src/index.jade

```diff
@@ -71,12 +71,15 @@ html
             .mui-textfield
               input.summary-picker__search(type="text", v-on:change="updateFilterSearch")
               label search
+            .mui-select.grouping
+              select(v-model="filterGroupSelection")
+                option(value="groupByNone") None
+                option(value="groupByRepos") Repo/Branch
+                option(value="groupByAuthors") Author
+              label group by
             .mui-select
               select(v-model="filterSort")
-                option(value="totalCommits") Contribution
-                option(value="variance") Variance
-                option(value="displayName") Author Name
-                option(value="searchPath") Repo/Branch Name
+                option(v-for="option in filterSortOptions" v-bind:value="option.value") {{ option.text }}
```

Comment:
just wondering, is it not possible to use `v-if` in such scenario?

Reply:
Yes, that is possible..